include/zephyr/bluetooth/audio/has.h

```diff
@@ -45,9 +45,9 @@ struct bt_has;
 
 /** Hearing Aid device type */
 enum bt_has_hearing_aid_type {
-	BT_HAS_HEARING_AID_TYPE_BINAURAL,
-	BT_HAS_HEARING_AID_TYPE_MONAURAL,
-	BT_HAS_HEARING_AID_TYPE_BANDED,
+	BT_HAS_HEARING_AID_TYPE_BINAURAL = BIT(0),
+	BT_HAS_HEARING_AID_TYPE_MONAURAL = BIT(1),
+	BT_HAS_HEARING_AID_TYPE_BANDED = BIT(2),
```

Comment:
Any reason why we are using `BIT(x)` instead of just `0x0X`? 

IMO using `BIT(X)` makes it seem like it is a bitfield (which it isn't).

Reply:
true - it's a bug (will change back to simple enum)